cfgov/v1/wagtail_hooks.py

```diff
@@ -42,28 +43,33 @@ def share(page, is_sharing, is_publishing):
     page.save()
 
 
-# Sets the latest revision with a state depending on if the request is to
-# publish, share, or save the page.
-def configure_page_revision(page, is_sharing, is_publishing):
+# `CFGOVPage.route()` will select the latest revision of the page where `live`
+# is set to True and return that revision as a page object to serve the request
+# so here we configure the latest revision to fall in line with that logic.
+#
+# This is also used as a signal callback when publishing in code or via
+# management command like publish_scheduled_pages.
+def configure_page_revision(page, is_publishing):
     latest = page.get_latest_revision()
     content_json = json.loads(latest.content_json)
     content_json['live'] = is_publishing
     content_json['shared'] = is_sharing or is_publishing
```

Comment:
Turns out when this was rebased [this line](https://github.com/cfpb/cfgov-refresh/pull/2189#discussion-diff-69183927R56) changed. Updating now.